handlers/java_handler/src/main/java/mercator/MavenUtils.java

```diff
@@ -166,17 +163,39 @@ public static boolean resolvePomsEnabled() {
         return Boolean.parseBoolean(System.getenv("MERCATOR_JAVA_RESOLVE_POMS"));
     }
 
+    private static Boolean ignoreDescription(Document pomDocument)  {
+        XPathFactory xpf = XPathFactory.newInstance();
+        XPath xpath = xpf.newXPath();
+        XPathExpression xpe = null;
+
+        try {
+            xpe = xpath.compile("/project/description");
+        } catch (javax.xml.xpath.XPathExpressionException e) {
+            e.printStackTrace();
```

Comment:
It's generally not a good idea to just print the stack trace and then continue as nothing happened :)

I also think that both try-catch blocks in this method could be combined into one.

Reply:
I agree, but if you take a look at the rest of implementation we are doing nothing with these exceptions other than catching them :-(.